src/Wrapper/PropertyAware.php

```diff
@@ -45,6 +45,21 @@ public function processProperties(): void
             }
 
             $value = $this->getPropertyValue($property);
+            if (!$value && $property->getPropertyData()->getCast() == CastType::EMPTY) {
```

Comment:
I know in the old exporter we weren't strict about this, but we should change that, because it is a string comparison anyway

```suggestion
            if (!$value && $property->getPropertyData()->getCast() === CastType::EMPTY) {
```